src/pages/index.js

```diff
@@ -122,7 +114,7 @@ function openProfilePopup() {
   const { profileName, description } = userInfo.getUserInfo();
   profileTitleElement.value = profileName;
   profileDescriptionElement.value = description;
-  editProfileFormValidator.toggleButtonState();
+  formValidators["edit-profile-form"].resetValidation();
```

Comment:
Everything is correct here but...


* I can't open the modals with a form. See the errors http://joxi.ru/4Ak65neTVeQzV2 . It's because you need to add attribute `name` for any tag `form` in your `index.html`



So, for the profile form in line 63 you'll have this markup


```html

        <form class="modal__form" name="edit-profile-form" id="edit-profile-form">

```


After that it will work correctly